MAUI/Cartesian-Charts/Line.md

```diff
@@ -267,3 +267,120 @@ this.Content = chart;
 
 ![Spline types chart in MAUI Chart](Chart-types_images/maui_spline_types_chart.png)
 
+### Enable Marker
+
+A marker, also known as a symbol, is used to determine or highlight the position of the data point. To give marker support in the series, set the ShowMarkers property to true. The default value of ShowMarkers property is false. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+...
+ <chart:LineSeries XBindingPath="Year"
+                   YBindingPath="Percentage"
+                   ItemsSource="{Binding Data}"
+                   ShowMarkers="True"/>
+
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+CategoryAxis primaryAxis = new CategoryAxis();
+chart.XAxes.Add(primaryAxis);
+NumericalAxis secondaryAxis = new NumericalAxis();
+chart.YAxes.Add(secondaryAxis);
+
+LineSeries series = new LineSeries()
+{
+    XBindingPath = "Year",
+    YBindingPath = "Percentage",
+    ItemsSource = new ViewModel().Data,
+    ShowMarkers= true,
+ };
+chart.Series.Add(series);
+this.Content= chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Customizing marker support
+
+To enable marker support customization, create an instance of the series MarkerSettings property. 
```

Comment:
In order to change how the series markers appear, create an instance of the [MarkerSettings]() property.